docs/source/conf.py

```diff
@@ -385,6 +385,12 @@ def generate_weights_table(module, table_name, metrics, include_pattern=None, ex
 generate_weights_table(
     module=M.detection, table_name="detection", metrics=[("box_map", "Box MAP")], exclude_pattern="Keypoint"
 )
+generate_weights_table(
+    module=M.detection,
+    table_name="instance_segmentation",
+    metrics=[("box_map", "Box MAP"), ("mask_map", "Mask MAP")],
```

Comment:
IMO mean average precision is short handed as mAP and not MAP?  That's what I think pycocotools or other report?

Reply:
I've seen both used. No strong opinion from me